index.html

```diff
@@ -1,12 +1,201 @@
+					</a>
+					<ul class="main-nav__list">
+						<li class="main-nav__item">
+							<a href="about.html" class="main-nav__link">About</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="index.html" class="main-nav__link" id="home">Home</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Program</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Join</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Sponsor</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">News</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" id="tecno" class="main-nav__link">Techno campaign</a>
+						</li>
+					</ul>
+					<ul class="main-nav__list__mobile">
+						<li class="main-nav__item XIcon">
+							<img
+								src="images/home/XIcon.svg"
+								alt="close"
+								class="main-nav__link"
+							/>
+						</li>
+						<li class="main-nav__item">
+							<a href="about.html" class="main-nav__link">About</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="index.html" class="main-nav__link">Home</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Program</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Join</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Sponsor</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">News</a>
+						</li>
+					</ul>
+				</div>
+				<img
+					src="images/home/hamburgermenu.svg"
+					alt="burgermenu"
+					class="burger"
+				/>
+			</nav>
+		</header>
+		<main class="card flex-column">
+			<div class="card__header">
+				<h2 class="card__subtitle">
+					"Hi! Make Money while making the world a better place"
+				</h2>
+				<h1 class="card__title">
+					THE INTERNATIONAL TECHNOPRENEURSHIP <br />
+					SUMMIT 2022
+				</h1>
+			</div>
+			<div class="card__body">
+				<p class="card__text">
+					An opportunity to learn to do the best you can while getting good at
+					your passion and also earning your cool cash, anchored by brilliant
+					individuals from over 90 countries.
+				</p>
+			</div>
+			<div class="card__footer">
+				<p class="date">2022.12.01(THU) ~ 16(FRI)</p>
+				<p class="venue">@ National arts theatre, Iganmu, Lagos, Nigeria</p>
+			</div>
+		</main>
+		<section class="main-program flex-column">
+			<h4 class="main-program__heading">Main<span> Pro</span>gram</h4>
+			<article class="main-program__items">
+				<div class="main-program__item">
+					<img src="images/home/research.svg" alt="research" />
+					<h5>Research</h5>
+					<p>
+						Lorem ipsum, dolor sit amet consectetur adipisicing elit. Est
+						eligendi fugiat eum quasi autem cupiditate!
```

Comment:
[Optional] I suggest that you adjust the program cards' a little more by increasing the font size of the descriptions to be more readable. This also matches the design given. Also, notice that the background on the design guideline is a little bit transparent, revealing the section's background.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190327629-2abd2b7b-f4f7-4545-a607-2b8a5a257397.png)

Given design:
![image](https://user-images.githubusercontent.com/20554583/190327739-2fd62ac7-8dc8-4838-815b-18d4e27c9044.png)

Notice how the description takes about half of the card's width.

Reply:
And likewise this one 😃.
I request that you please check it once again and give me feedback.
Thanks 🙃